### DISCUSSION TITLE
Nightly build

### DISCUSSION THREAD

**Discussion by rgieseke** [orbitalquark/textadept#42] [General]
Is the nightly build "nightly" to fetch dependencies like Scintillua?
Otherwise one could have a build whenever a new commit (or group of commits) is pushed.

**Comment by orbitalquark** [comment]
It's a compromise between having bleeding edge builds and not wasting a ton of resources (electricity for builds and hammering download servers for 3rd-party dependencies). The latter is mitigated to an extent with the "textadept-build" repo, but there are still a handful of requests made.

**Comment by rgieseke** [comment]
I see. If you want to trigger them (for Scintillua changes) one could apparently do something like described here:
https://github.community/t/triggering-by-other-repository/16163
